src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
The active state for the stars is not being applied correctly. The `:nth-child(-n + 4)` selector will only apply the active state to the first 4 stars of every `.stars` block, regardless of the rating. You should move this logic to your JavaScript, where you can dynamically apply the active state based on the actual rating.